pkg/storage/sds/models.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sds
```

Comment:
I'm slightly confused about the split between the sds package and the edv package.

Aren't these models related to the edv portion of the sds spec?

Reply:
Indeed they are... that was an error on my part. I've moved models inside the `edv` package.